man/tpm2_clear.1.md

```diff
@@ -1,24 +1,21 @@
-tpm2_clear 1 "DECEMBER 2017" tpm2-tools
-==================================================
```

Comment:
I loved the comment about the previous committer :rofl:

My guess is that you wrote this tool when were still using md2man instead of pandoc and forgot to change the format during a rebase?